app/services/create_induction_tutor.rb

```diff
@@ -24,14 +24,12 @@ def call
         InductionCoordinatorProfile.create!(user: user, schools: [school])
       end
 
-      # TODO: This should really be using deliver_later, but this can't be tested via Cypress
-      # After discussion leaving this as deliver_now with  this comment
```

Comment:
Have we resolved this?

Reply:
Now it is :) All the emails are being delivered inline, while all other jobs are being delayed.